homeworks/roman.strokan_Strokan/homework_7/task1.js

```diff
@@ -0,0 +1,17 @@
+function fib(array, endVal) {
```

Comment:
by the task you are trying to solve, you don't need a starting array parameter for your function, it needs only to accept the final value

Reply:
you already know that starting value supposed to be 0